carmajava/signal_plugin/src/main/java/gov/dot/fhwa/saxton/carma/signal_plugin/TrafficSignalPlugin.java

```diff
@@ -634,33 +678,82 @@ public void loop() throws InterruptedException {
 
     /**
      * Helper function to evaluate state variables for the stopping condition
+     *
      */
     private void evaluateStatesForStopping() {
```

Comment:
Please add a comment both in the function description and in the loop function which describes that this method can result in Thread.Sleep being called